app/containers/AccountProvider/saga.js

```diff
@@ -165,6 +174,51 @@ export function* isAvailableAction(isValid) {
   yield call(isValid);
 }
 
+const getInvitedTable = async eosService => {
```

Comment:
getTableRows can search record by primary key. Pass user name into the method.

```const invitedUsers = await eosService.getTableRows(
    INVITED_USERS_TABLE,
    INVITED_USERS_SCOPE,
    account,
    process.env.EOS_TOKEN_CONTRACT_ACCOUNT,
  );
```

Reply:
Done